aip/general/0203.md

```diff
@@ -44,42 +44,51 @@ document this behavior for clients.
 
 ## Vocabulary
 
-### Required
+### Identifier
 
-The use of `REQUIRED` indicates that the field **must** be present (and set to
-a non-empty value) on the request or resource.
+The use of `IDENTIFIER` indicates that the field is used to identify the
+resource. It must be attached to the name parameter and any other fields that
+are used in identifying the resource within the collection. See [fields
+representing resource names].
```

Comment:
Is this to say we want to see it used outside of messages that represent resources? If so, couldn't that conflict with other field_behavior annotations?

The sentence following kind of contradicts the second part of this first sentence...

Reply:
Are you referring to the reference to "fields representing resource names"? That's just the name of the section, although really that section could be called "resource names within resources" or something like that.

I linked to the section to try to ensure we don't have redundant (and therefore hard to maintain) guidance.

I tried to reword the section altogether to call out fields within a resource used to identify the resource. PTAL.

Reply:
I was originally thinking we may want to account for some older services like compute, but based on what I'm seeing, they don't really have field identifiers.

even older services like bigquery still have name: https://github.com/googleapis/googleapis/blob/d994e780c732da0a902070d0314ca401fb6132df/google/cloud/bigquery/reservation/v1/reservation.proto#L1120-L1121.

So I'll just update the wording that it should be exclusively for name, as that's the longterm guidance anyway.

Reply:
Thanks, I see what you are saying now and I agree with your analysis.

Reply:
ok updated, PTAL.

Reply:
Yep, looks good, thanks!